controller/engine_controller.go

```diff
@@ -1090,7 +1091,7 @@ func syncWithRestoreStatus(log logrus.FieldLogger, engine *longhorn.Engine, rsMa
 
 	for _, status := range rsMap {
 		if status.Error != "" {
-			log.WithError(errors.New(status.Error)).Warn("need to wait for the restore error handling before the restore invocation")
+			log.WithError(errors.New(status.Error)).Warn("needing to wait for the restore error handling before the restore invocation")
```

Comment:
`Waiting for the restore error handling before the restore invocation`

Reply:
Modified.